db/db.js

```diff
@@ -28,37 +28,39 @@ const songSchema = mongoose.Schema({
 
 const Song = mongoose.model('Song', songSchema);
 
-console.log
-
-for(let i = 0; i < data.length; i++) {
-  const songDocument = new Song(data[i])
-  songDocument.save((error, data) => {
+const getSong = (id, callback) => {
+  Song.find({ id: id }).exec((error, data) => {
     if (error) {
-      console.log('Issue saving data', error);
+      console.log('There was an error looking up song in db', err);
     } else {
-      console.log('Import successful', data);
+      console.log('Song found!', data);
+      callback(null, data[0]);
     }
   });
 }
 
-Song.find((error, songs) => {
+const getAll = (callback) => {
+  Song.find({ id: 15 }).exec((error, data) => {
```

Comment:
Is there a reason why the ID is set to 15, or is that just a placeholder for now?

Reply:
Oh that is a mistake, I'll change it